tests/pr_test.go

```diff
@@ -17,6 +17,23 @@ const resourceGroup = "geretain-test-mongo"
 const defaultExampleTerraformDir = "examples/default"
 const autoscalingExampleTerraformDir = "examples/autoscale"
 const completeExampleTerraformDir = "examples/complete"
+const fsCloudTerraformDir = "examples/fscloud"
```

Comment:
Is there much difference between fscloud and complete? Probably the same? Can we remove complete example and just maintain the fscloud example and run tests against that?